client/shell/mdb_code_versioning.py

```diff
@@ -17,10 +17,10 @@ def version(version_config):
     session = requests.Session()
     session.headers['Authorization'] = 'token %s' \
         % version_config[constants.ACCESS_TOKEN_KEY]
-    
+
     # check if the repo exists, if not create it
     repo_exists_url = 'https://api.github.com/repos/%s/%s' % \
-        (version_config[constants.GIT_USERNAME_KEY], 
+        (version_config[constants.GIT_USERNAME_KEY],
```

Comment:
`ConfigUtils.ConfigReader`'s methods return exactly what they did before, so version_config is identical to the previous version, and this continues to function. 

(ConfigUtils.ConfigReader ConfigConstants to define _outputs_. I didn't want to touch that, since I wasn't sure what parts of the program, like this one, used them, or how.)